packages/webapp/pages/onboarding.tsx

```diff
@@ -233,10 +292,20 @@ export function OnboardPage(): ReactElement {
       return;
     }
 
+    let targetId = onboardingV2;
```

Comment:
v2 no longer exists, we can clean it up :)

Reply:
Will remove that and v3.